src/index.js

```diff
@@ -97,50 +97,64 @@ function flags () {
     tokenContract: config.tokenContract,
     communityContract: config.communityContract
   }
-  devLog('flags', flags_)
+  debugLog('flags', flags_)
   return flags_
 }
 
 // Start elm app with flags
 const app = Elm.Main.init({
   flags: flags()
 })
+Sentry.addBreadcrumb({
+  message: 'Started Elm app',
+  level: 'info',
+  type: 'debug',
+  category: 'started',
+  data: {
+    flags: flags()
+  }
+})
 
 // Register Service Worker After App
 registerServiceWorker()
 
 // Log
 
-function devLog (name, arg) {
+function debugLog (name, arg) {
   if (env === 'development') {
     console.log('[dev]', name, arg)
   }
+
+  Sentry.addBreadcrumb({
+    message: typeof arg === 'string' || arg instanceof String ? arg : name,
+    level: 'info',
+    type: 'debug'
+  }
+  )
```

Comment:
I've changed this function that is used a lot in this file to be a general log. Sometimes the arg contain sensible data, so if its not a string, use the name.